modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTable.scala

```diff
@@ -128,6 +129,12 @@ private sealed trait SequenceTableBuilder[S: Eq, D <: DynamicConfig: Eq]
     extends SequenceTableDefs[D]:
   private type Props = SequenceTable[S, D]
 
+  private def scrollIfNeeded(element: dom.Element) = Callback {
```

Comment:
Do we have this function in explore too? maybe eventually move it to lucuma-ui

Reply:
Yeah we have it in one place